crates/chain/src/tx_graph.rs

```diff
@@ -1107,6 +1245,9 @@ pub struct ChangeSet<A = ()> {
     pub anchors: BTreeSet<(A, Txid)>,
     /// Added last-seen unix timestamps of transactions.
     pub last_seen: BTreeMap<Txid, u64>,
+    /// Added timestamps of when a transaction is last evicted from the mempool.
+    #[cfg_attr(feature = "serde", serde(default))]
```

Comment:
Is this `cfg_attr` only needed to load data from a pre-v2 serialized file?

Reply:
Yes, this ensures backwards compatibility of our serialized changeset (as we promised).